sdk/src/test/scala/com/horizen/account/block/AccountBlockHeaderTest.scala

```diff
@@ -0,0 +1,173 @@
+package com.horizen.account.block
+
+import com.horizen.account.fixtures.{AccountBlockFixture, ForgerAccountGenerationMetadata}
+import com.horizen.fixtures.CompanionsFixture
+import com.horizen.params.{MainNetParams, NetworkParams}
+import com.horizen.proof.VrfProof
+import com.horizen.proposition.VrfPublicKey
+import com.horizen.secret.VrfSecretKey
+import com.horizen.utils.BytesUtils
+import com.horizen.validation.InvalidSidechainBlockHeaderException
+import com.horizen.vrf.VrfGeneratedDataProvider
+import org.junit.Assert.{assertArrayEquals, assertEquals, assertTrue, fail => jFail}
+import org.junit.Test
+import org.scalatestplus.junit.JUnitSuite
+
+import java.io.{BufferedReader, BufferedWriter, FileReader, FileWriter}
+import scala.util.{Failure, Success}
+
+class AccountBlockHeaderTest extends JUnitSuite with CompanionsFixture with AccountBlockFixture {
+  val vrfGenerationDataSeed = 178
+  val vrfGenerationPrefix = "SidechainBlockHeaderTest"
+  //set to true for update vrf related data
+  if (false) {
+    VrfGeneratedDataProvider.updateVrfProof(vrfGenerationPrefix, vrfGenerationDataSeed)
+    VrfGeneratedDataProvider.updateVrfSecretKey(vrfGenerationPrefix, vrfGenerationDataSeed)
+  }
+
+  val vrfKeyPair: Option[(VrfSecretKey, VrfPublicKey)] = {
+    val secret: VrfSecretKey = VrfGeneratedDataProvider.getVrfSecretKey(vrfGenerationPrefix, vrfGenerationDataSeed)
+    val publicKey: VrfPublicKey = secret.publicImage()
+    Option((secret, publicKey))
+  }
+
+  val vrfProofOpt: Option[VrfProof] = Option(VrfGeneratedDataProvider.getVrfProof(vrfGenerationPrefix, vrfGenerationDataSeed))
+  val header: AccountBlockHeader = createUnsignedBlockHeader(123L, vrfKeyPair, vrfProofOpt)._1
+  val params: NetworkParams = MainNetParams()
+
+  @Test
+  def serialization(): Unit = {
+    val bytes = header.bytes
+
+
+    // Test 1: try to deserializer valid bytes
+    val serializedHeaderTry = AccountBlockHeaderSerializer.parseBytesTry(bytes)
+    assertTrue("AccountBlockHeader expected to by parsed.", serializedHeaderTry.isSuccess)
```

Comment:
typo: "by parsed" instead of "be parsed". Same below

Reply:
Done